### ISSUE TITLE
2E Sheet?

### ISSUE THREAD

**Issue by tbsvttr** [TomCosaert/FoundryVTT-ICRPG#4]
Thanks a lot for this great system plugin!

I do not understand to what version of the ICRPG rules the 2E sheet is referring to. Both my printed book (by Amazon) and my PDF (by DriveThroughRPG) are labeled as Second Edition (and look like the second edition), but they have character sheets that differ from the 2E sheet in this plugin, but are the same as the normal sheet. The exact differences are with the base armor class (you can not put anything in there in the 2E sheet), abilities (you have them in the 2E sheet), and the mastery (you have it in the 2E sheet).

Maybe this is me not understanding something with the rules or the plugin or there is some miscommunication. I report this so other are as confused as me.

**Comment by TomCosaert** [comment]
**TL;DR** The ```2E``` suffix refers to the version of the Quickstart booklet, i.e.
* ```icrpg.IcrpgCharacterSheet``` = ICRPG 2E + Quickstart rules
* ```icrpg.IcrpgCharacterSheet2E``` = ICRPG 2E + [Quickstart **2nd Edition**](https://www.drivethrurpg.com/product/221038/INDEX-CARD-RPG-Free-Quickstart) rules

I understand your confusion and will try to clarify.

For starters, this module offers no support for ICRPG 1E whatsoever, but only for **ICRPG 2E** and its accompanying **Quickstart** booklet. 

However, a few months back, Hankerin [released](https://forums.runehammer.online/t/updated-quickstart-availible/4403/2) a free, updated [Quickstart **2nd Edition**](https://www.drivethrurpg.com/product/221038/INDEX-CARD-RPG-Free-Quickstart) booklet, containing many ideas originally planned for ICRPG 3E, and shelved the release of ICRPG 3E. I heartily recommend reading it, as it contains some awesome stuff!

So that's what the ```2E``` suffix refers to. In hindsight, I should have called the new (default) sheet ```icrpg.IcrpgCharacterSheetQS2E``` or ```icrpg.IcrpgCharacterSheetQuickStart``` to reflect the current situation a bit better, but I didn't ;-)

Hope this helps.
